report/sections/system-design.tex

```diff
@@ -34,6 +35,39 @@ \subsection{Model}
   \caption{High-level state machine of the system.}
   \label{fig:state_machine}
 \end{figure*}
+\subsubsection{Mobile platform}
+The model used in this case is the unicycle model, due to the difrential steering. This is because of the mobile platform has only two wheels/trucks and it is not able to apply any steering angel to it's wheels. The only way this robot can change orientation is by giving different velocity on each wheel-driving servo on left- and right- side. With this feature it is also possible to change the orientation of the mobile platform without changing the position of the platform. I.e. the robot is able to spin while the right-hand sidewheels have the same velocity as the left-hand side wheels, in opposite direction. 
+
+\begin{figure}[h!]
+\centering
+\includegraphics[width=0.4\textwidth]{sections/assets/car-unicycle.png}
+\caption{Unicycle model of a car-like robot. $v_L$ and $v_R$ represent the left- and right-hand side wheels' velocity respectively. The robot follows a path around the instantaneous center of rotation (ICR) where $R_L$ and $R_R$ are the distances from left and right wheels to ICR respectively and $w$ is the distance between them.}
+\label{fig:UnicycleModel}
+\end{figure} 
+
+As shown in Fig. \ref{fig:UnicycleModel} The robot follows a curved path with the instantaneous center of rotation at its center. The left-hand side wheels have velocity $v_L$ and moves along an arc with radius $R_L$ during the time that right-hand side wheels moves along another arc with radius $R_R$ at the speed of $v_R$.The turning rate of the body is
```

Comment:
Change `Fig. \ref{fig:UnicycleModel}` to `Fig.~\ref{fig:UnicycleModel}` so that there isn't unecessary much whitespace between `Fig.` and its the number in rendered form.